homeassistant/components/freebox/device_tracker.py

```diff
@@ -12,8 +14,7 @@
 async def async_get_scanner(hass, config):
     """Validate the configuration and return a Freebox scanner."""
     scanner = FreeboxDeviceScanner(hass.data[DATA_FREEBOX])
-    await scanner.async_connect()
-    return scanner if scanner.success_init else None
```

Comment:
Why are these lines removed and why is `async_connect` not used?

Reply:
these 2 lines are the root cause of the problem. As we check if the freebox is there. If not, we return None and then scanner is never recreated by home assistant.
Returning a not None scanner will force HA to try to update. Error checking is done on device update instead